src/explore-education-statistics-common/src/components/Code.module.scss

```diff
@@ -0,0 +1,36 @@
+@import '~govuk-frontend/dist/govuk/base';
+
+.container {
+  position: relative;
+
+  &:active.copyButton,
+  &:focus-within.copyButton,
+  &:hover.copyButton {
+    opacity: 1;
+  }
+}
+
+.copyButton {
+  max-width: fit-content;
+  position: absolute;
+  right: govuk-spacing(2);
+  top: govuk-spacing(2);
+
+  &:active {
+    top: 12px;
+  }
+}
+
+.pre {
+  border: 1px solid #bfc1c3;
+  background: #f9f9f8;
+  max-width: 100%;
+  overflow-x: auto;
+  padding: 60px 15px 15px !important;
```

Comment:
What predefined style is being overridden here? Maybe check with a more dedicated FE, but I typically try to avoid using `!important` unless absolutely necessary, opting instead to rely on the cascade, or specificity.

Reply:
Unfortunately the package is setting an inline `style` on the element, so this is the only way overwrite that.  
There may be a way to configure the styles, but I think that may come out in the wash when we want to render more code snippets (more than just 1 or 2 lines)